src/downloadmanager.cpp

```diff
@@ -166,21 +176,6 @@ bool DownloadManager::moveMyAppPackage(QString path)
     return true;
 }
 
-QString DownloadManager::aptoideApk(QString packageName)
-{
-    QString apkPath("/data/app/");
-    QString aptoideApk = QString("%1/%2.apk").arg(apkPath, packageName);
-    if (!QFile(aptoideApk).exists()) {
```

Comment:
I don't see any reason for this kind of lightweight Aptoide installed check.

As ```apkd-launcher com.aptoide.partners.jolla_tablet_store-1.apk``` works without path we should not have path there.